.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - run: python -m pip install 'tox<4'
       - run: tox -e checkformatting
   Lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-32-cores
```

Comment:
As in LMS we're also using 32-cores for lint because pylint also runs in parallel jobs.